setup.py

```diff
@@ -71,17 +71,19 @@ def _unavailable(self, e):
 py_impl = getattr(platform, 'python_implementation', lambda: None)
 is_pypy = py_impl() == 'PyPy'
 is_jython = 'java' in sys.platform
-is_pure = 'PURE_PYTHON' in os.environ
```

Comment:
Thing is, this is the only check for 'PURE_PYTHON' in the codebase.  If you remove it and don't add a runtime check, it becomes a no-op.  I don't think we want that.